### DISCUSSION TITLE
v1.76.0

### DISCUSSION THREAD

**Discussion by alextran1502** [immich-app/immich#3911] [Announcements]
# v1.76.0

## Highlights

Welcome to release `v1.76.0` of Immich. There are many improvements in this version that you will like. Some of the key highlights include:

- Map view on mobile
- Customizable machine learning settings

### Map view on mobile

Thanks to @shalong-tanwen's amazing work, the map view is now available on mobile. You can see the location of your photos and videos on the map. This is a great way to see where your memories are located and where the events are happening.
Search Page | Map view | With Selection | Light-mode
:-: | :-: | :-: | :-:
![search_page](https://github.com/immich-app/immich/assets/139912620/0ad10dba-1cbf-4e01-af4e-e5048e7e320e) | ![map_view](https://github.com/immich-app/immich/assets/139912620/7581ea3c-7146-4b03-ad06-847c8a83882e) | ![with_selection](https://github.com/immich-app/immich/assets/139912620/5c5d7ee7-6001-4c69-a872-7b02c41de2af) | ![light_mode](https://github.com/immich-app/immich/assets/139912620/d247d86d-d24b-4361-8871-d28bf18a1963)

### Customizable machine learning settings
Thanks to the continuous improvment to machine learning pipe line from @mertalev, you now have more control of the machine learning settings, including the score we use to determine if a face is present in a photo or how the face grouping works. You can now customize the settings to your liking.
![image](https://github.com/immich-app/immich/assets/27055614/bd9aec69-1668-4a27-a923-1130a3b3bcda)

- - - -
## Support

<p align="center">
<img src="https://media.giphy.com/media/LStqgGESXW8XnuCv5y/giphy.gif" width="250" title="SUPPORT THE PROJECT!"> 
</p>

If you find the project helpful, you can support Immich via the following channels.

- Monthly donation via [GitHub Sponsors](https://github.com/sponsors/alextran1502)
- One-time donation via [GitHub Sponsors](https://github.com/sponsors/alextran1502?frequency=one-time&sponsor=alextran1502)
- [Librepay](https://liberapay.com/alex.tran1502/)
- [buymeacoffee](https://www.buymeacoffee.com/altran1502)
- Bitcoin: 1FvEp6P6NM8EZEkpGUFAN2LqJ1gxusNxZX

It is a great way to let me know that you want me to continue developing and working on this project for years to come.

### Web
* chore(web): Fixing up missing awaits by @samip5 in https://github.com/immich-app/immich/pull/3882
* fix(web, mobile): camera info by @waclaw66 in https://github.com/immich-app/immich/pull/3904
* fix(web): null location by @waclaw66 in https://github.com/immich-app/immich/pull/3906
* feat(web): customizable ML settings by @mertalev in https://github.com/immich-app/immich/pull/3891
* chore(web): Refer to FFmpeg documentation by @daniele-athome in https://github.com/immich-app/immich/pull/3908

### Mobile
* feat(mobile): map view by @shalong-tanwen in https://github.com/immich-app/immich/pull/3661
* chore(mobile): add more catch all error log detail by @alextran1502 in https://github.com/immich-app/immich/pull/3893

### Server
* fix(server): Fixes immich-admin because npm bin does not like arguments by @danieldietzler in https://github.com/immich-app/immich/pull/3864
* chore(deps): bump docker/setup-buildx-action from 2.9.1 to 2.10.0 by @dependabot in https://github.com/immich-app/immich/pull/3897
* chore(server): bump server dependencies by @alextran1502 in https://github.com/immich-app/immich/pull/3899
* fix(server): set stream index for ffmpeg command by @mertalev in https://github.com/immich-app/immich/pull/3903

**Full Changelog**: https://github.com/immich-app/immich/compare/v1.75.2...v1.76.0

<hr /><em>This discussion was created from the release <a href='https://github.com/immich-app/immich/releases/tag/v1.76.0'>v1.76.0</a>.</em>

**Comment by DrSpaldo** [comment]
Are you doing roadmaps for future releases?

**Comment by pkampkam** [comment]
Hi, I cannot see the new map view on mobile. I've got Android and the app comes from F-droid. Is machine-learning needed for mobile view?

**Reply by shenlong-tanwen** [reply]
> Hi, I cannot see the new map view on mobile. I've got Android and the app comes from F-droid. Is machine-learning needed for mobile view?

The F-droid build is failing due to a Non-Free dependency used for the Map View. A package called `geolocator` is used to get the current user's location for one particular feature of the map view which is causing the issue. I'm hoping to fix it before the next release.

**Reply by pkampkam** [reply]
Thanks!
P.S. On mobile, all my pics show 0x0 pixel size, while file size in MB is shown properly. Is that also F-droid thing, or is only me?

**Reply by pkampkam** [reply]
Hi! While wainting for the F-droid app, I saw there was 1.77 available, which is not true anymore. There in only 1.75.2 again.
The store version is 1.78.1 ... Is it possible we have F-droid version with Map View? Regards.

**Reply by shenlong-tanwen** [reply]
> Hi! While wainting for the F-droid app, I saw there was 1.77 available, which is not true anymore. There in only 1.75.2 again. The store version is 1.78.1 ... Is it possible we have F-droid version with Map View? Regards.

The latest version in F-droid is [1.77](https://f-droid.org/repo/app.alextran.immich_100.apk) which has the Map view. F-droid updates are usually slower than the store updates. The meta in F-droid repo points to the latest release (1.78.1). It might get updated in the next F-droid build cycle
